aggregator_core/src/datastore.rs

```diff
@@ -710,6 +710,34 @@ impl<C: Clock> Transaction<'_, C> {
         Ok(())
     }
 
+    /// Sets or unsets the expiration date of a task.
+    #[tracing::instrument(skip(self))]
```

Comment:
nit: this should have the lower event level set too
```suggestion
    #[tracing::instrument(skip(self, err(level = Level::DEBUG)))]
```